packages/remote-replay-launcher/src/resource-tracker.ts

```diff
@@ -0,0 +1,65 @@
+import os from "os";
+import { TestRun } from "@alwaysmeticulous/client";
+import * as Sentry from "@sentry/node";
```

Comment:
I guess Sentry is already initialized whether called [from the CLI](https://github.com/alwaysmeticulous/meticulous-sdk/blob/main/packages/cli/src/main.ts#L31) or from [reports diff action](https://github.com/alwaysmeticulous/report-diffs-action/blob/main/src/actions/cloud-compute/cloud-compute.ts#L14) so this works.